task_03.js

```diff
@@ -0,0 +1,6 @@
+// Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), 
+// и выводит сумму равную n + nn + nnn, где n не перемножаются, а конкатенируются
+
+let n = 2
```

Comment:
Check task 2/3 files for this like variables created with let, that are not changed further and replace let with const